tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -165,37 +145,27 @@ public function setLastPublicShareData(SimpleXMLElement $responseXml): void {
 	}
 
 	/**
-	 * @return SimpleXMLElement
-	 * @throws Exception
+	 * @param string $sharer
+	 * @param SimpleXMLElement $shareData
+	 *
+	 * @return void
 	 */
-	public function getLastShareData():SimpleXMLElement {
-		return $this->getLastShareDataForUser($this->userWhoCreatedLastShare);
+	public function addToCreatedUserGroupShares(string $sharer, SimpleXMLElement $shareData): void {
+		$this->createdUserGroupShares["$sharer"] = ["shareData" => $shareData];
```

Comment:
or simply can be this
```suggestion
		$this->createdUserGroupShares["$sharer"] = $shareData;
```